indico/modules/rb/operations/bookings.py

```diff
@@ -349,15 +349,17 @@ def create_booking_for_event(room_id, event):
         return None
 
 
-def get_active_bookings(limit, start_dt, last_reservation_id=None, **filters):
+def get_active_bookings(limit, start_dt, end_dt=None, last_reservation_id=None, **filters):
     criteria = [ReservationOccurrence.start_dt > start_dt]
```

Comment:
Not sure if it's here or else where, but I think there's a case where a booking starting shortly before the event is not shown at all.

Here's my test data:

- event on 24 April 2025, 09:00 - 11:00
- booking on Thu 24/04/2025, 08:00 → 12:00
- another test booking on Fri 25/04/2025,08:00 → 12:15

With the newly added toggle disabled, no bookings are shown (good!).
With the toggle enabled, I only get my second text booking (and other bookings from April 25th onwards).

![image](https://github.com/user-attachments/assets/e032ce1b-fb4a-4552-8b20-fdf5ccbb6ae8)

Reply:
That's part of #6568

> As a summary, this is what I think it could still be addressed in this issue:
> 
> Currently, the bookings page uses the current date. It'd be better if it used the date of the event (or the first day in case of multi-day events)

Reply:
The current code shows occurrences during the same day, but not previous days. I obviously don't mind changing it to the previous 1-2-3 days, but I think the intention of that issue is not to get a never-ending list of bookings until you reach the event's date.

Reply:
But it wasn't a previous day in this case... so it should have shown the 24th, no?

Reply:
Oh yes, my bad. I mixed the dates. It should be fixed with [201c713](https://github.com/indico/indico/pull/6846/commits/201c713c03235c7290660b2a5bff363083619828).